src/azure-cli/azure/cli/command_modules/signalr/_params.py

```diff
@@ -67,6 +70,9 @@ def load_arguments(self, _):
     with self.argument_context('signalr cors remove') as c:
         c.argument('allowed_origins', options_list=['--allowed-origins', '-a'], nargs='*', help='space separated origins that should be allowed to make cross-origin calls (for example: http://example.com:12345). To allow all, use "*"')
 
+    with self.argument_context('signalr cors update') as c:
+        c.argument('allowed_origins', options_list=['--allowed-origins', '-a'], nargs='*', help='space separated origins that should be allowed to make cross-origin calls (for example: http://example.com:12345). To allow all, use "*"')
+
     # Network Rule
```

Comment:
Please overwrite id_part property to None in `list` commands to resolve linter errors. For example

```python
    with self.argument_context('signalr cors list') as c:
        c.argument('signalr_name', id_part=None)
```